src/psyclone/psyGen.py

```diff
@@ -4307,14 +4327,23 @@ def search_subroutine(nodelist, searchname):
             raise InternalError("Unexpected kernel AST. Could not find "
                                 "subroutine: {0}".format(name))
 
+        arg_list = []
```

Comment:
Please add a comment here, e.g. 'save list of kernel arguments'

Reply:
Done